images/pulumi/configs/latest.apko.yaml

```diff
@@ -37,33 +31,46 @@ contents:
     - pulumi-language-java
     - glibc-locale-en
     - openjdk-17-jre
-    - openjdk-17-default-jvm
```

Comment:
intentional?

Reply:
cc @joshrwolf

Reply:
it was removed from another image, checking to see..

Reply:
reverted. this removes need for JAVA_HOME